pkgs/applications/backup/vorta/default.nix

```diff
@@ -72,7 +79,9 @@ python3Packages.buildPythonApplication rec {
     ''
       export HOME=$(mktemp -d)
       export FONTCONFIG_FILE=${fontsConf};
-      # For tests/test_misc.py::test_autostart
+    ''
+    # For tests/test_misc.py::test_autostart
```

Comment:
What's this change for?

Reply:
I agree with this change.

Reply:
I think this makes it unnecessarily hard to read and rebuilds are not a problem with leaf packages like this.

Reply:
Hard to read or not, subjective or not, I believe that those comments shouldn't end up in the final derivation. But OK, since there is no established rule in the project about this, I would not consider this as a blocker and move on.